src/services/character/character.service.ts

```diff
@@ -13,10 +13,13 @@ export class CharacterService {
     return CharacterService._instance
   }
 
-  public async getEntity(id: string):Promise<Document | null> {
+  public async getEntity(id: string):Promise<Document | void> {
     return await characters.findOne({ ownerId: id }, (err: Error, characterFound) => {        
       if (characterFound) return characterFound
-      else return console.log(`Error: Character not found because: ${err}`)
+    })
+    .catch((err): void => {
+      console.log(`Error: Character not found because: ${err}`)
```

Comment:
I thought you were keeping the error throw

Reply:
Hmm... Nope, because the bot would crash and I don't want the bot to crash at character creation.

Reply:
But you can catch the error!

Reply:
To do that you just have to add .catch() after calling getEntity().
So getEntity().catch().